app/views/layouts/application.html.erb

```diff
@@ -1,16 +1,57 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Blog</title>
+    <title>My Blog </title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag "styles.css"%>
```

Comment:
what is the function of `stylesheet_link_tag`?

Reply:
Returns a link tag to the implemented Bootstrap theme style.